[2025-09-17T01:58:04.474+0000] {processor.py:161} INFO - Started process (PID=173) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:58:04.480+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T01:58:04.483+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:58:04.483+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:58:04.519+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:58:04.513+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T01:58:04.521+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:58:04.644+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.186 seconds
[2025-09-17T01:58:35.633+0000] {processor.py:161} INFO - Started process (PID=210) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:58:35.635+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T01:58:35.640+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:58:35.638+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:58:35.656+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:58:35.653+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T01:58:35.656+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:58:35.759+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.143 seconds
[2025-09-17T01:59:06.960+0000] {processor.py:161} INFO - Started process (PID=238) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:59:06.967+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T01:59:06.973+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:59:06.971+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:59:07.005+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:59:06.998+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T01:59:07.009+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:59:07.196+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.245 seconds
[2025-09-17T01:59:38.176+0000] {processor.py:161} INFO - Started process (PID=264) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:59:38.192+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T01:59:38.196+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:59:38.195+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:59:38.241+0000] {logging_mixin.py:188} INFO - [2025-09-17T01:59:38.227+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T01:59:38.243+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T01:59:38.415+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.256 seconds
[2025-09-17T02:00:09.062+0000] {processor.py:161} INFO - Started process (PID=289) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:00:09.106+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:00:09.111+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:00:09.110+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:00:09.151+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:00:09.144+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:00:09.152+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:00:09.345+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.305 seconds
[2025-09-17T02:04:08.848+0000] {processor.py:161} INFO - Started process (PID=173) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:04:08.853+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:04:08.858+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:04:08.857+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:04:08.887+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:04:08.880+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:04:08.894+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:04:08.992+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.168 seconds
[2025-09-17T02:04:39.998+0000] {processor.py:161} INFO - Started process (PID=211) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:04:40.001+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:04:40.012+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:04:40.007+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:04:40.037+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:04:40.033+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:04:40.040+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:04:40.152+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.174 seconds
[2025-09-17T02:05:11.227+0000] {processor.py:161} INFO - Started process (PID=237) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:05:11.247+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:05:11.269+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:05:11.264+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:05:11.627+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:05:11.594+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:05:11.631+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:05:11.835+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.651 seconds
[2025-09-17T02:05:42.366+0000] {processor.py:161} INFO - Started process (PID=262) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:05:42.369+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:05:42.371+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:05:42.371+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:05:42.387+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:05:42.384+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:05:42.388+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:05:42.444+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.086 seconds
[2025-09-17T02:06:13.316+0000] {processor.py:161} INFO - Started process (PID=287) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:06:13.319+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:06:13.323+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:06:13.322+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:06:13.345+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:06:13.343+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:06:13.346+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:06:13.458+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.153 seconds
[2025-09-17T02:06:43.890+0000] {processor.py:161} INFO - Started process (PID=311) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:06:43.894+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:06:43.898+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:06:43.897+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:06:43.922+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:06:43.913+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:06:43.930+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:06:44.013+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.133 seconds
[2025-09-17T02:07:15.474+0000] {processor.py:161} INFO - Started process (PID=337) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:07:15.485+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:07:15.490+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:07:15.487+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:07:15.552+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:07:15.535+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:07:15.557+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:07:15.757+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.317 seconds
[2025-09-17T02:07:46.539+0000] {processor.py:161} INFO - Started process (PID=372) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:07:46.546+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:07:46.548+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:07:46.547+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:07:46.588+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:07:46.583+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:07:46.593+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:07:46.727+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.220 seconds
[2025-09-17T02:08:17.616+0000] {processor.py:161} INFO - Started process (PID=397) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:08:17.619+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:08:17.621+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:08:17.621+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:08:17.637+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:08:17.631+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:08:17.637+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:08:17.696+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.089 seconds
[2025-09-17T02:08:48.229+0000] {processor.py:161} INFO - Started process (PID=422) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:08:48.260+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:08:48.288+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:08:48.280+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:08:48.447+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:08:48.444+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:08:48.448+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:08:48.621+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.412 seconds
[2025-09-17T02:09:19.320+0000] {processor.py:161} INFO - Started process (PID=444) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:09:19.337+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:09:19.346+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:09:19.344+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:09:19.377+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:09:19.372+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:09:19.378+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:09:19.531+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.232 seconds
[2025-09-17T02:09:50.689+0000] {processor.py:161} INFO - Started process (PID=469) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:09:50.695+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:09:50.700+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:09:50.697+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:09:50.950+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:09:50.900+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:09:50.955+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:09:51.251+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.616 seconds
[2025-09-17T02:10:22.677+0000] {processor.py:161} INFO - Started process (PID=512) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:10:22.682+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:10:22.685+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:10:22.684+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:10:22.717+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:10:22.707+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:10:22.719+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:10:22.871+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.212 seconds
[2025-09-17T02:10:53.981+0000] {processor.py:161} INFO - Started process (PID=555) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:10:53.991+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:10:53.999+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:10:53.997+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:10:54.024+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:10:54.020+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:10:54.025+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:10:54.084+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.117 seconds
[2025-09-17T02:11:40.090+0000] {processor.py:161} INFO - Started process (PID=187) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:11:40.093+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:11:40.096+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:11:40.095+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:11:40.114+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:11:40.111+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:11:40.115+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:11:40.214+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.138 seconds
[2025-09-17T02:12:10.477+0000] {processor.py:161} INFO - Started process (PID=212) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:12:10.481+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:12:10.484+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:12:10.483+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:12:10.506+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:12:10.500+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:12:10.508+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:12:10.618+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.166 seconds
[2025-09-17T02:12:42.208+0000] {processor.py:161} INFO - Started process (PID=237) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:12:42.306+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:12:42.390+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:12:42.386+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:12:42.613+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:12:42.561+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:12:42.620+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:12:42.905+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.939 seconds
[2025-09-17T02:13:13.891+0000] {processor.py:161} INFO - Started process (PID=261) to work on /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:13:13.896+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/equities_intraday_stream_dag.py for tasks to queue
[2025-09-17T02:13:13.911+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:13:13.905+0000] {dagbag.py:538} INFO - Filling up the DagBag from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:13:13.995+0000] {logging_mixin.py:188} INFO - [2025-09-17T02:13:13.976+0000] {dagbag.py:348} ERROR - Failed to import: /opt/airflow/dags/equities_intraday_stream_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 344, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/equities_intraday_stream_dag.py", line 4, in <module>
    from scripts.equities_intraday_producer import run_equities_intraday_producer
ModuleNotFoundError: No module named 'scripts.equities_intraday_producer'
[2025-09-17T02:13:14.001+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/equities_intraday_stream_dag.py
[2025-09-17T02:13:14.147+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/equities_intraday_stream_dag.py took 0.281 seconds
